gsrs-fda-substance-extension/src/main/java/fda/gsrs/substance/exporters/ClinicalTrialEuropeDTOExporter.java

```diff
@@ -0,0 +1,221 @@
+package fda.gsrs.substance.exporters;
+
+import gov.hhs.gsrs.clinicaltrial.europe.api.ClinicalTrialEuropeDTO;
+import gov.hhs.gsrs.clinicaltrial.europe.api.ClinicalTrialEuropeDrugDTO;
+import gov.hhs.gsrs.clinicaltrial.europe.api.ClinicalTrialEuropeMeddraDTO;
+import gov.hhs.gsrs.clinicaltrial.europe.api.ClinicalTrialEuropeProductDTO;
+import gsrs.module.substance.SubstanceEntityService;
+import ix.ginas.exporters.*;
+import ix.ginas.models.v1.Substance;
+
+import java.io.IOException;
+import java.util.*;
+
+enum ClinicalTrialEuropeDefaultColumns implements Column {
+    TRIAL_NUMBER,
+    TITLE,
+    SUBSTANCE_NAME,
+    SUBSTANCE_KEY,
+    CONDITIONS,
+    SPONSOR_NAME,
+    RESULTS
+    }
+
+public class ClinicalTrialEuropeDTOExporter implements Exporter<ClinicalTrialEuropeDTO> {
+
+    private final Spreadsheet spreadsheet;
+
+    private int row = 1;
+
+    private final List<ColumnValueRecipe<ClinicalTrialEuropeDTO>> recipeMap;
+
+    private static SubstanceEntityService substanceEntityService;
```

Comment:
This shouldn't be static if you're setting it in a constructor ... unless something really weird is going on.